app/assets/stylesheets/steps.scss

```diff
@@ -6,6 +6,7 @@ $light-grey: #f5f5f5;
 $light-grey-2: #f8f8f8;
 $black: #0B0C0C;
 $red: #b10e1e;
+$grey: #eee;
```

Comment:
This isn't really important, but for consistency with other colours we might want to use the full code? And decide on caps/no caps.

Reply:
@vanitabarrett Is the full code #eeeeee?

Reply:
@leenagupte Yep!